projects/frontend/data-pipelines/gui/projects/ui/src/assets/data/appConfig.json

```diff
@@ -1,42 +1 @@
-{
-  "auth": {
-    "skipAuth": false,
-    "consoleCloudUrl": "https://console-stg.cloud.vmware.com/",
-    "orgLinkRoot": "/csp/gateway/am/api/orgs/",
-    "authConfig": {
-      "issuer": "https://console-stg.cloud.vmware.com/csp/gateway/am/api/",
-      "redirectUri": "$window.location.origin/",
-      "skipIssuerCheck": true,
-      "requestAccessToken": true,
-      "oidc": true,
-      "strictDiscoveryDocumentValidation": false,
-      "clientId": "8qQgcmhhsXuhGJs58ZW1hQ86h3eZXTpBV6t",
-      "responseType": "code",
-      "scope": "openid ALL_PERMISSIONS customer_number group_names",
-      "showDebugInformation": true,
-      "silentRefreshRedirectUri": "$window.location.origin/silent-refresh.html",
-      "useSilentRefresh": true,
-      "silentRefreshTimeout": 5000,
-      "timeoutFactor": 0.25,
-      "sessionChecksEnabled": true,
-      "clearHashAfterLogin": false,
-      "logoutUrl": "https://console-stg.cloud.vmware.com/csp/gateway/discovery?logout",
-      "nonceStateSeparator": "semicolon"
-    },
-    "resourceServer": {
-      "allowedUrls": [
-        "https://console-stg.cloud.vmware.com/",
-        "https://gaz-preview.csp-vidm-prod.com/",
-        "/data-jobs"
-      ],
-      "sendAccessToken": true
-    },
-    "refreshTokenConfig": {
-      "start": 500,
-      "remainingTime": 360,
-      "checkInterval": 60
-    }
-  },
-  "ignoreComponents": [],
-  "ignoreRoutes": []
-}
```

Comment:
I don't like this approach, because it was easier to fetch the default appConfig.json file and modify it (before the change), and now there is no sampled JSON a user can use OOTB -since hardcoded.

Reply:
Is appConfig supposed to be available to the end user?